src/VisualStudio/NuGet.Packaging.VisualStudio/Templates/Projects/PortableClassLibrary/PortableClassLibrary.csproj

```diff
@@ -32,18 +32,18 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <PropertyGroup Label="Configuration">
-    <Id>$safeprojectname$</Id>
-    <Version>1.0.0</Version>
+    <PackageId>$safeprojectname$</PackageId>
+    <PackageVersion>1.0</PackageVersion>
```

Comment:
Should add the same [targets-based mechanism](https://github.com/NuGet/NuGet.Build.Packaging/blob/dev/src/VisualStudio/NuGet.Packaging.VisualStudio/Templates/Projects/Package/Package.targets#L4-L9) for versioning this project too